terraform/azure/outputs.tf

```diff
@@ -109,8 +109,8 @@ output "bastion_public_ip" {
 resource "local_file" "ansible_inventory" {
   content = templatefile("inventory.tmpl",
     {
-      hana-name           = module.hana_node.cluster_nodes_name[0],
-      hana-pip            = module.hana_node.cluster_nodes_public_ip[0],
+      hana-name           = module.hana_node.hana_name[0],
```

Comment:
So we are going to use internally the names already used in the Ansible API (the inventory.yaml) ... right?

Reply:
Yeah iirc.